src/Homepage/Homepage.module.scss

```diff
@@ -1,3 +1,7 @@
 .homepage {
     background-color: red;
+}
+
+h2{
+    margin: 0px;
 }
```

Comment:
This will impact all `h2`s on the site...
is that what you want?

if it isn't => then could you replace with a classname.
if it is => then could you not put it in a module - CSS here should only impact the HomePage Component. `index.scss` would be a better place.

Reply:
Done